### PR TITLE
Update UG formatting

### PR DESCRIPTION
**Pull request by YHWong20**
* Added CSS styling for callout boxes.
* Added `[` and `]` to wrap optional arguments/flags for commands
* Committed sample images for mac - getting started section (not added to UG yet)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -650,28 +700,31 @@ Unsure as to how you can save your BinBash data? Don't worry! Your data is autom
...
+|-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| **add**     | `add -re -n ITEM_NAME -d ITEM_DESCRIPTION -s SALE_PRICE -c COST_PRICE [-e EXPIRY_DATE] [-q ITEM_QUANTITY] [-t THRESHOLD]` <br> `add -op -n ITEM_NAME -d ITEM_DESCRIPTION -c COST_PRICE [-e EXPIRY_DATE] [-q ITEM_QUANTITY] [-t THRESHOLD]`                          | Adds a new item to the inventory.                                                                                     |
+| **search**  | `search -n NAME_QUERY -d DESCRIPTION_QUERY -q QUANTITY_RANGE -c COST_PRICE_RANGE -s SALE_PRICE_RANGE -e EXPIRY_DATE_RANGE -l NUMBER_OF_RESULTS`                                                                                                                     | Searches for items in the inventory based on various criteria.                                                        |
```
Comment (nur-haziq) [inline docs/UserGuide.md:720]: Thanks for updating this in the 'Command Summary' section.

We can consider restating the point on placeholders within square brackets [ ] being optional. 

This is for power users who jumped straight to this section, and didn't read the note in the 'Features' section

File: docs/UserGuide.md [hunk 2]
```diff
@@ -68,9 +68,17 @@ can make the most out of BinBash. Let's get your inventory organized efficiently
...
+</div>
 
-> ℹ️ This is an info box that may appear in the guide. Any additional information about a feature will be detailed in sections like this one here.
+<div id="infoCallout" style="padding: 1em; border: 0 solid #9ec1cf;border-left-width: 4px;border-radius: 6px; margin-top: 1rem; margin-bottom: 1rem; padding: 1em; border-radius: 4px; color: #293132; background-color: #eef9fc;">
+ℹ️ This is an info box that may appear in the guide. Any additional information about a feature will be detailed in sections like this one here.
+</div>
+
+<div id="tipCallout" style="padding: 1em; border: 0 solid #9ee09e;border-left-width: 4px;border-radius: 6px; margin-top: 1rem; margin-bottom: 1rem; padding: 1em; border-radius: 4px; color: #293132; background-color: #e6f5e6;">
+💡 This is a tip box that may appear in the guide. Things that might be useful for you can be found in sections like this one here.
+</div>
```
Comment (nur-haziq) [inline docs/UserGuide.md:81]: Thanks for adding this!

**Review by nur-haziq — APPROVED** [review]
LGTM. Thanks for implementing this useful stylistic change.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -175,16 +183,15 @@ Now, let's take a look at the flags you can use:
...
+ℹ️ <strong>Note:</strong>
+<li>Only one item type flag can be specified for each item. This means that you can only use either <code>-re</code> or <code>-op</code> but not both at the same time. </li> 
+<li>The <code>-e</code> flag should be provided if the item that you are adding is a Perishable item. That is to say, it will expire by the provided expiry date.</li>
+<li>The <code>-s</code> flag should be provided if the item that you are adding is a Retail item. This means that this item is meant to be sold.</li>
+<li>The flags can be placed in any order. There is no specific order that you have to abide by.</li>
+<li>Words in <code>UPPER_CASE</code> are the arguments that are meant to be supplied by you.<br>For example, in <code>-n ITEM_NAME</code>, <code>ITEM_NAME</code> would represent the name of the item you are adding (e.g., <code>add -n apple</code>).</li>
+<li>For some commands, if flags and arguments are wrapped in square brackets, they are optional.<br>For example, <code>add [-q ITEM_QUANTITY]</code> signify that the <code>-q</code> flag, as well as its argument, are optional for the command.</li>
+</div>
```
[the remark this answers is not in the document]
Comment (YHWong20) [reply docs/UserGuide.md:194]: Got it, I've reintroduced this in the `Flags` section.

File: docs/UserGuide.md (hunk 1 above)
Comment (YHWong20) [reply docs/UserGuide.md:720]: Sure! I'll add this in.

**Pull request merged**